src/arithmetic/algebraic_expression/algebraic_expression_construction.c

```diff
@@ -70,7 +70,7 @@ static bool _AlgebraicExpression_ContainsVariableLengthEdge
 	case AL_OPERAND:
 		if(exp->operand.edge) {
 			QGEdge *e = QueryGraph_GetEdgeByAlias(qg, exp->operand.edge);
-			return QGEdge_VariableLength(e);
+			return QGEdge_RangeLength(e);
```

Comment:
Given the comment on `qg_edge.h` I believe we can revert this change back to its original version.